includes/class-wc-gateway-dintero-hp.php

```diff
@@ -1140,11 +1123,25 @@ private function capture( $order, $transaction = null ) {
 				$items = $this->get_items_to_capture_embed($order);
 			}
 
+			$transaction_amount = $transaction['amount'];
+
+			// We experience inexplicable differences of one cent.
+			// To lessen the impact, we will allow difference of one cent,
```

Comment:
not actually a cent, its the smalles unit of the currency

Reply:
clarified the comment